hardware_enablement/kmm-kernel-module-management.adoc

```diff
@@ -16,6 +16,47 @@ include::modules/kmm-installation.adoc[leveloffset=+1]
 include::modules/kmm-installing-using-web-console.adoc[leveloffset=+2]
 include::modules/kmm-installing-using-cli.adoc[leveloffset=+2]
 include::modules/kmm-installing-older-versions.adoc[leveloffset=+2]
+include::modules/kmm-deploying-modules.adoc[leveloffset=+1]
+include::modules/kmm-creating-module-cr.adoc[leveloffset=+2]
+include::modules/kmm-security.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../authentication/understanding-and-managing-pod-security-admission.adoc[Understanding and managing pod security admission].
```

Comment:
Re: the build error, all of these xrefs need an anchor ID added (with the value of the `:context:` for the assembly they reference). I don't know what they are for these files, but e.g.
```suggestion
* xref:../authentication/understanding-and-managing-pod-security-admission.adoc#my-cool-anchor-id[Understanding and managing pod security admission].
```

Reply:
@jeana-redhat That worked. Thx!

Reply:
/label merge-review-needed